Make no assumptions about the content of the scan note free text

annehaley opened the pull request:
Instead of trying to parse the user and creation time of a decision from the unformatted text of the note column, this PR makes no assumptions about that content and copies it to the import file as-is. MIQA will then fill in its own creation time at the time of import and the creator can be null. We just have to hope that the decision-maker signed their initials in the free text in order to identify who made the decision.

Here is a screenshot of one such decision after being imported:
![image](https://user-images.githubusercontent.com/44912689/175310616-8014212f-ed69-402f-9bbc-dbf63d25d1cf.png)

kipohl commented:
- I do not understand why the value of 
```    quality_translation["questionable"] = "0"  ``` was changed - those are the values of xnat probably just need more explanation s
- the other changes look good other than the print statement - and please nowhere ever erase any scan notes

annehaley commented:
I removed the print statements (nice catch) and I moved the conversion of the decision code strings. If the `0, 1, 2` format is needed by xnat, we can just interpret those codes for MIQA in the `miqa_file_generation` module. So I added a dictionary mapping there. If the decision string for a scan isn't one of "0", "1", or "2", though, the code will default to questionable ("Q?"). This shouldn't be a problem because `check_new_sessions` is the one generating those strings, but this is built in as a catch-all. Let me know if this is not desired behavior. 

I tried not to lose any scan note information. Is there somewhere in the code that concerns you in regards to this? Perhaps I can explain better in some comments.
